Allow serving static assets from `baseDir` (#108)
fastify/fastify-swagger-ui: README.md

Note that this behavior is disabled by default in `@fastify/swagger-ui`.

### Bundling

> If you want to bundle Swagger UI with your application, you need to copy the swagger-ui static files to your server yourself and set the `baseDir` option to point to your folder.
> 

Reviewer: ```suggestion
To bundle Swagger UI with your application, the swagger-ui static files need to be copied to the server and the `baseDir` option set to point to the file directory.

```
